packages/gallery/src/components/gallery/proGallery/slideshowView.js

```diff
@@ -660,37 +663,34 @@ class SlideshowView extends React.Component {
     const arrowsBaseClasses = [
       'nav-arrows-container',
       useDropShadow ? 'drop-shadow' : '',
+      utils.isMobile() ? ' pro-gallery-mobile-indicator' : '',
     ];
-    return [
-      hideLeftArrow ? null : (
-        <button
-          className={
-            arrowsBaseClasses.join(' ') +
-            (utils.isMobile() ? ' pro-gallery-mobile-indicator' : '')
```

Comment:
In Master only the left arrow has this class. Im not sure why but the change you make places this class on right and left

Reply:
try to figure out what this class is. It looks like someone just missed adding it on the other arrow (added it only once).